README.md

```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://api.travis-ci.org/kpn/pyrandall.svg?branch=master)](https://travis-ci.org/kpn/pyrandall)
+
 # PYRANDALL
 
 Simulator and Sanitytest framework that encourages to write tests upfront, but are suitable to run as e2e tests.
```

Comment:
for consistency please change sanitytest to validation

Reply:
Will open a new PR with update to documentation such as README.md. And include the other minor changes we discussed.